lessons/pydata/databases/index.ipynb

```diff
@@ -0,0 +1,1078 @@
+{
```

Comment:
<p>Tady by stálo za to aspoň krátce shrnout, co vlastně SQLAlchemy dělá a proč nám to tady pomůže.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/79/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>